bpf/flows_sock.c

```diff
@@ -243,6 +263,10 @@ int socket__http_filter(struct __sk_buff *skb) {
             bpf_ringbuf_submit(record, 0);
         }
     }
+    // finally, when flow receives FIN or RST, clean flow_directions
```

Comment:
Same comment here about the possibility of not cleaning up on ring buffer alloc fail.

Reply:
done